masterlist.yaml

```diff
@@ -8589,12 +8589,21 @@ plugins:
 
   - name: 'Arcanum.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/23488/' ]
-    msg: [ *patch3rdPartySRPC ]
+    msg:
+      - *patch3rdPartySRPC
+      - <<: *alsoUseX
+        subs: [ '[Arcanum - A New Age of Magic 4.0.1 Bugfix](https://www.nexusmods.com/skyrimspecialedition/mods/46322)' ]
+        condition: 'checksum("Arcanum.esp", AA67A039)'
+      - type: warn
+        content: 
+          - lang: en
+            text: 'The BSA files included with this version of Arcanum need to be renamed in order for the mod to work. Remove "4.0 BSA" from the name of each .bsa file so you are left with Arcanum.bsa and Arcanum - Textures.bsa.'
```

Comment:
I added something similar to this to the Oblivion masterlist.
So it could be used instead as it has some translations.

```yaml

  - &renameXtoY
    type: warn
    content:
      - lang: en
        text: 'You need to rename %1% to %2% or this mod won''t work.'
      - lang: de
        text: 'Sie müssen %1% zu %2% umbenennen, ansonsten funktioniert diese Mod nicht.'
      - lang: ja
        text: '%1%を%2%にリネームしないと機能しません。'
      - lang: ru
        text: 'Вам нужно переименовать %1% в %2% для подключения ресурсов мода..'
```
```yaml
  - name: 'MigHorseArmor.esp'
    msg:
      - <<: *renameXtoY
        subs:
          - 'HRMHorseArmor.bsa'
          - 'MigHorseArmor.bsa'
        condition: 'not file("MigHorseArmor.bsa") and file("HRMHorseArmor.bsa")'
```